packages/apps/spaces/app/layout.tsx

```diff
@@ -153,7 +153,7 @@ export default async function RootLayout({
           }}
         />
 
-        <body className='scrollbar'>
```

Comment:
Thank you !!!

I think this was a change done by @acalinica when he initially made the branch. I have looked now at the scrollbar and it seems to be the same.


I thought about creating wrapper components for Flex, Stacks but with tailwind it seems redundant. 
For example  a VStack  can be replaced with a `<div className="space-y-4" />` instead of `<VStack spacing='4' />` .

I also think this can reduce the amount of imports that VStack, HStack, Flex, Box  add to the bundle size